auth/src/main/java/org/aerogear/mobile/security/checks/RootedCheck.java

```diff
@@ -23,4 +24,9 @@ public SecurityCheckResult test(final Context context) {
         final RootBeer rootBeer = new RootBeer(context);
         return new SecurityCheckResultImpl(this, rootBeer.isRooted());
     }
+
+    @VisibleForTesting
+    SecurityCheckResult test(final Context context, final RootBeer rootBeer) {
```

Comment:
I don't really like the `visiblefortesting`.
I think it would be nicer as in [#2](https://github.com/ciaranRoche/aerogear-android-sdk/pull/2).
@wei-lee @matzew @secondsun WDYT?

Reply:
I am fine w/ that annotation - it's part of the standard android class lib.

However, the proposed solution does indeed read much nicer, than using the annotation

Reply:
I agree that it is part of the standard android class lib, however I don't like it anyway ;)
With the proposed approach in [#2](https://github.com/ciaranRoche/aerogear-android-sdk/pull/2) you have the advantage that you test the real method, instead of testing a method created ad-hoc for testing. Moreover, with the current approach we will have to maintain two methods instead of one.

Reply:
for this use case, I think we should go with the PR as you suggested. But sometimes we do have to use `visiblefortesting` as it is just a lot simpler.